views/migration/teradata/README.md

```diff
@@ -0,0 +1,22 @@
+# Teradata Views
+
+This directory contains views that mimic the behavior of tables / views in Teradata.
+
+## SYS_CALENDAR.CALENDAR
```

Comment:
Add link to the view file

Reply:
Also add a description about the view and mention that the columns of the view are described below

Reply:
Done.